src/bun.js/bindings/bindings.cpp

```diff
@@ -100,6 +100,11 @@
 #include "JavaScriptCore/InternalFieldTuple.h"
 #include "wtf/text/StringToIntegerConversion.h"
 
+static WTF::StringView StringView_slice(WTF::StringView sv, unsigned start, unsigned end)
+{
+    return sv.substring(start, end - start);
+}
```

Comment:
This is a great helper. Do we have a WTF like place for it. (... guessing this may be it)